src/Mailjet/Message/CommitteeCitizenInitiativeOrganizerNotificationMessage.php

```diff
@@ -38,11 +38,11 @@ private static function getTemplateVars(
         string $initiativeName
     ): array {
         return [
-            'referent_firstname' => self::escape($referentFirstName),
-            'referent_lastname' => self::escape($referentLastName),
-            'referent_contact_link' => $contactLink,
+            'animator_firstname' => self::escape($referentFirstName),
+            'animator_lastname' => self::escape($referentLastName),
+            'animator_contact_link' => $contactLink,
             'committee_name' => self::escape($committeeName),
-            'target_firstname' => self::escape($targetFirstName),
+            'prenom' => self::escape($targetFirstName),
```

Comment:
There is no need of this variable here, isn't it ?
You should create `getRecipientVars` for it.